assets/js/front/epoch-front-compiled.js

```diff
@@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
              *
              * @since 0.0.1
              */
-            $( app.form_el ).submit( function( event ) {
+            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
```

Comment:
Will do. Changes are fairly minor.

Can I skip the script concat on the next one and just do the unminified?
On Tue, Sep 1, 2015 at 00:42 Josh Pollock <notifications@github.com> wrote:

> In assets/js/front/epoch-front-compiled.js
> <https://github.com/postmatic/epoch/pull/64#discussion_r38387163>:
>
> > @@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
> >               *
> >               * @since 0.0.1
> >               */
> > -            $( app.form_el ).submit( function( event ) {
> > +            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
>
> @ronalfy <https://github.com/ronalfy> At this point it is outdated. It
> will need a new PR against 1.0.3-dev. Sorry about that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/postmatic/epoch/pull/64/files#r38387163>.
>